packages/2018-neighborhood-development/src/components/UnderstandingStaffCuts/index.js

```diff
@@ -1,25 +1,147 @@
+  getUnderstandingStaffCutsData,
+} from '../../state/understanding-staff-cuts/selectors';
 
 export class UnderstandingStaffCuts extends React.Component {
   componentDidMount() {
-    // initialize data here
+    this.props.init();
   }
 
   render() {
+    const {
+      isLoading,
+      error,
+      understandingStaffCuts,
+    } = this.props;
+
+    const tableCols = [
+      {
+        align: 'left',
+        header: 'School',
+        key: 'school',
+      },
+      {
+        header: 'Students per FTE Staff',
+        key: 'student-staff',
+        columns:
+        [
+          {
+            align: 'right',
+            header: 'PPS Proposed 2018-2019 Staffing',
+            key: 'students',
+          },
+          {
+            align: 'right',
+            header: 'Hack Oregon Predicted Staffing',
+            key: 'predictions',
+          },
+          {
+            align: 'right',
+            header: 'Variation',
+            key: 'variation',
+          },
+        ],
+      },
+      {
+        align: 'right',
+        header: '% Change Enrollment',
+        key: 'changes_in_enrollment_pct',
+      },
+      {
+        header: 'Student Demographics',
+        key: 'students-demo',
+        columns:
+        [
+          {
+            align: 'right',
+            header: '% From Historically Underserved Groups',
+            key: 'historically_underserved_pct',
+          },
+          {
+            align: 'right',
+            header: '% Recieving Free Meals by Direct Certification',
+            key: 'free_meals_direct_cert_pct',
+          },
+        ],
+      },
+    ];
+
+    const tableData = {
+      columns: tableCols,
+      data: understandingStaffCuts,
+    };
+
     return (
       <CivicStoryCard
         title="Understanding Staff Cuts in Portland Public Schools"
         slug="understanding-staff-cuts"
+        loading={isLoading}
+        error={error && 'Could not load sweeps data'}
       >
-        <Placeholder issue="201" />
+        <Collapsable>
+          <Collapsable.Section>
+            <div>
+              <p>Portland Public Schools (PPS) parents received an email explaining proposed staffing changes in mid-April, 2018. Hack Oregon conducted an analysis of these changes using PPS’s stated goals listed below.</p>
+              <ul>
+                <li>Ensure schools have adequate staffing to maintain reasonable class sizes.</li>
+                <li>Support the district’s equity goals.</li>
+                <li>Provide baseline academic program offerings at every school.</li>
+                <li>Provide greater stability over time.</li>
+              </ul>
+              <p>See the results of our analysis in the sortable table below.</p>
+            </div>
+          </Collapsable.Section>
+          <Collapsable.Section hidden>
+            <div>
+              { understandingStaffCuts &&
+                <DataTable
+                  data={tableData}
+                />
+              }
```

Comment:
Seems weird for the data component to be hidden by default.

Reply:
Yeah, it's definitely a bit off. It was a workaround to avoid having to implementing pagination or some other method of limiting the length the DataTable component, because as is it's something like three screens long, which seemed a little much for a single card.

Perhaps that's something that could be implemented in collapsable - something like a max height for a section that would cut off mid-component. That would require some creative reworking of collapsable, and I'm not sure if that's the right approach.